simplq/src/styles/joinPage.module.scss

```diff
@@ -1,36 +1,15 @@
 @import 'utils';
 
-.header {
-  @include queue-name-header();
-}
-
-.join-button {
-  @include center-horizontally();
-}
-
-.input {
-  @include center-horizontally();
-  width: 50%;
-  fieldset {
-    border-radius: 0.5rem;
-  }
-}
-
 .form {
-  @include center-horizontally();
-  table {
-    width: 50%;
-  }
-  td {
-    padding-bottom: 2rem;
-  }
-}
-
-.main {
-  min-height: 100vh;
-}
-
-.message {
-  @include center-horizontally();
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 56vh;
```

Comment:
Hi. Very sorry. Can you change to 35rem instead?

Reply:
ya sure.